engine/apps/integrations/mixins/ratelimit_mixin.py

```diff
@@ -37,6 +40,23 @@ def get_rate_limit_per_team_key(_, request):
     return str(request.alert_receive_channel.organization_id)
 
 
+def get_rate_limit(group, request):
+    custom_ratelimits = settings.CUSTOM_RATELIMITS
+
+    organization_id = str(request.alert_receive_channel.organization_id)
+
+    if group == RATELIMIT_INTEGRATION_GROUP_NAME:
+        if organization_id in custom_ratelimits:
+            return custom_ratelimits[organization_id]["integration"]
+        return RATELIMIT_INTEGRATION
+    elif group == RATELIMIT_TEAM_GROUP_NAME:
+        if organization_id in custom_ratelimits:
+            return custom_ratelimits[organization_id]["organization"]
+        return RATELIMIT_TEAM
+    else:
+        raise Exception("Unknown group")
+
```

Comment:
👆 this allows to specify custom ratelimit for integrations api in settings and env variable